Change 28597733 by wdavis@wdavis-118a6c on 2020/06/19 10:23:22

        nscq: fix SONAME assignment during build.

        This was incorrectly using only the major version,
        rather than the full libnvidia-nscq.so.$(MAJOR_VERSION).

        Bug 3030420: NSCQ: incorrect soname used

        #review-28597104
        Review link: https://p4sw-swarm.nvidia.com/reviews/28597104
        Reviewed by: aalsudani

        PRESUBMIT_TESTING:
        DVS_SANITY: http://builds4u/dvs/#/change/2859685650237503.3?eventType=Virtual

Jobs fixed ...

3030420-2006 on 2020/06/23 by p4dtg *open*

        NSCQ: incorrect soname used

Affected files ...

... //sw/dev/gpu_drv/chips_a/apps/nvswitch/nscq/unix.nvmk#6 edit

Differences ...

--- //sw/rel/gpu_drv/r450/r451_40/apps/nvswitch/nscq/unix.nvmk        2020-06-18 05:02:30.000000000 -0700
+++ unix.nvmk   2020-06-18 05:02:30.000000000 -0700


@@ -21,2 +21,4 @@
 
+NSCQ_SO = libnvidia-nscq.so
+
 BIN_LDFLAGS += -static-libstdc++ -static-libgcc
@@ -25,6 +27,6 @@
 BIN_LDFLAGS += -Wl,-lpthread
-BIN_LDFLAGS += -Wl,-soname -Wl,$(NSCQ_API_VERSION_MAJOR)
+BIN_LDFLAGS += -Wl,-soname -Wl,$(NSCQ_SO).$(NSCQ_API_VERSION_MAJOR)
 BIN_LDFLAGS += -Wl,--version-script=$(NSCQ_VERSION_SCRIPT)
 
-NSCQ_LIB = $(OUTPUTDIR)/libnvidia-nscq.so
+NSCQ_LIB = $(OUTPUTDIR)/$(NSCQ_SO)
 NSCQ_LIB_IMPL = $(NSCQ_LIB).$(NV_VERSION_STRING)
